components/modules/blocks/Hero/Shapes-Hero.tsx

```
@@ -0,0 +1,329 @@
+      animate={{ opacity: 1, scale: 1 }}
+      transition={{ duration: 1.5, delay, ease: "easeOut" }}
+      className={cn("absolute", className)}
+    >
+      <motion.div
+        animate={{
+          y: [0, -20, 0],
+          x: [0, 10, 0],
+          opacity: [0.4, 0.8, 0.4],
+        }}
+        transition={{
+          duration: 8 + Math.random() * 4,
```

ESLint

[error] Error: Cannot call impure function during render

`Math.random` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/components/modules/blocks/Hero/Shapes-Hero.tsx:104:25
  102 |         }}
  103 |         transition={{
> 104 |           duration: 8 + Math.random() * 4,
      |                         ^^^^^^^^^^^^^ Cannot call impure function
  105 |           repeat: Number.POSITIVE_INFINITY,
  106 |           ease: "easeInOut",
  107 |         }}

(react-hooks/purity)